test/api/version/apiVersion.controller.test.ts

```diff
@@ -71,11 +90,44 @@ beforeEach(() => {
   );
 });
 
+it('returns a 200 and apiVersion data but does not show s3 published status', async () => {
```

Comment:
```suggestion
it('APIVersionController.getAPIVersionsForAPI() returns a 200 and apiVersion data but does not show s3 published status', async () => {
```

Marc introduced this new pattern of not using `describe` statements and including "<class>.<method>()" at the beginning of each test description